src/TestSuite/TestCase.php

```diff
@@ -195,6 +197,33 @@ public function loadFixtures(): void
         }
     }
 
+    /**
+     * Load routes for the application.
+     *
+     * If no application class can be found an exception will be raised.
+     * Routes for plugins will *not* be loaded. Use `loadPlugins()` or use
+     * `Cake\TestSuite\IntegrationTestCaseTrait` to better simulate all routes
+     * and plugins being loaded.
+     *
+     * @param array|null $appArgs Constuctor parameters for the application class.
+     * @return void
+     * @since 4.0.1
+     */
+    public function loadRoutes(?array $appArgs = null): void
+    {
+        $appArgs = $appArgs === null ? [CONFIG] : $appArgs;
```

Comment:
The trailing slash from `CONFIG` needs to be removed as all methods of `BaseApplication` that use `$this->configDir` again add a slash.

https://github.com/cakephp/cakephp/blob/9886fa7dcef259a1ff497a5b8fdf739401499c89/src/Http/BaseApplication.php#L139